platform/pulpcore/tasking/celery_app.py

```diff
@@ -126,9 +128,9 @@ def initialize_worker(sender, instance, **kwargs):
     removed along with the setting of online=False on the worker record in the database itself.
     The working directory specified in /etc/pulp/server.conf (/var/cache/pulp/<worker_name>)
     by default is removed and recreated. This is called early in the worker start process,
-    and later when it's fully online, pulp_celerybeat will discover the worker as usual to allow
-    new work to arrive at this worker. If there is no previous work to cleanup, this method
-    still runs, but has no effect on the database.
+    and later when it's fully online, workers will discover the worker as usual to allow new work to
```

Comment:
Not strictly true, the worker will write to the database and the resource manager will "discover" it itself by checking the db.  There isn't really any discovery happening on the worker, just checks to make sure there aren't zero of them.